app/views/layouts/admin.html.haml

```diff
@@ -31,9 +31,10 @@
       #admin-dashboard-sidebar.admin-dashboard-sidebar.position-left.off-canvas.off-canvas-absolute.reveal-for-medium.data-off-canvas
         .admin-dashboard-sidebar-title-area
           .admin-dashboard-open-sidebar
-            %button#open-sidebar.admin-dashboard-open-sidebar-button.show-for-medium{'aria-label': "open menu", type: "button"}
-              %span{'aria-hidden': "true"}
-                %a{href: "#"}
+            %button.admin-dashboard-open-sidebar-button
+            .show-for-medium{ 'aria-label': "open menu", type: "button" }#open-sidebar
```

Comment:
i don't think this wrapped line is the same thing at all.
it was a button with a class of "show-for-medium" -- and it's changed to a button with nothing in it and a div after it.

Reply:
![image](https://user-images.githubusercontent.com/12266/29999287-9bd0d788-9096-11e7-88e0-837141ce2ddf.png)

It'll need fixing - it'll break admin if we release this

Reply:
Proposed a fix in #69